.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
Can you please add the `auto` value to the `text` prop? 🙏 

```bash
* text=auto eol=lf
```

The `auto` tells to git to automatically identify text files, avoiding to edit binaries files, etc.